### PR TITLE
Test Maya fix for MAYA-122305 (center pivot)

### PR DESCRIPTION
**Pull request by ppt-adsk**
Relies on an unreleased version of Maya, which doesn't yet have a version label, so cannot be merged yet.

### CODE REVIEW

File: test/lib/ufe/testComboCmd.py [hunk 1]
```diff
@@ -890,6 +891,56 @@ def testCenterPivotUndo(self):
...
+
+        checkNullPivotsAndCompensations()
+
+        # Rotate around x axis so that moving the pivot will create a rotate
+        # pivot translation as compensation, which keeps the total object
+        # transformation constant.
+        cmds.rotate(45, 0, 0, r=True, os=True, fo=True)
+
+        # Move the pivot.
+        cmds.move(0, 5, 0, r=True, urp=True, usp=True)
```
Comment (pierrebai-adsk) [inline test/lib/ufe/testComboCmd.py:928]: I'm looking at the docs here: https://help.autodesk.com/cloudhelp/2023/CHS/Maya-Tech-Docs/CommandsPython/move.html and cannot see what urp and usp flags are. What are their long names? Pretty sure they must be pivot-related :)

**Review by pierrebai-adsk — APPROVED** [review]
LGTM, the short flags make the behavior of the code harder to understand though.

File: test/lib/ufe/testComboCmd.py (hunk 1 above)
Comment (ppt-adsk) [reply test/lib/ufe/testComboCmd.py:928]: Yeah, unfortunately undocumented, which we should fix, ufeRotatePivot, ufeScalePivot.

**Comment by ppt-adsk** [comment]
Been over a month, will rebase on dev.

**Pull request closed without merging**